xbmc/addons/GUIDialogAddonInfo.cpp

```diff
@@ -86,6 +87,15 @@ bool CGUIDialogAddonInfo::OnMessage(CGUIMessage& message)
       }
       if (iControl == CONTROL_BTN_INSTALL)
       {
+        if (m_localAddon)
+        {
+          if (m_localAddon->Type() == ADDON_ADSPDLL && ActiveAE::CActiveAEDSP::Get().IsProcessing())
+          {
+            CGUIDialogOK::ShowAndGetInput(24077, 0, 24085, 0);
+            return true;
+          }
+        }
+
```

Comment:
It should be up to date. I just rebased his work, that we can merge it and then improve it.

You're right the labels are missing in the strings.po. I don't know 24077 and 24085 should be :(

@AlwinEsch: can you help?